.github/workflows/dart.yml

```
@@ -1,31 +1,52 @@
+    needs: build
+    runs-on: ubuntu-latest
+    # Condition to run only on pull requests that are not in draft
+    if: github.event.pull_request.draft == false
+    steps:
+      - name: Merge PR
+        uses: pascalgn/automerge-action@v0.16.3
+        env:
+          # The GITHUB_TOKEN is required for the action to interact with the repo
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          # Optional: Define merge strategy (merge, squash, rebase)
+          MERGE_METHOD: "squash"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)